deploy/k3s/refresh_creds.yaml

```diff
@@ -0,0 +1,37 @@
+# Cron job to refresh our local credentials
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: refresh-creds
+spec:
+  schedule: "0 */2 * * *"  # Every 2 hours
+  jobTemplate:
+    spec:
+      backoffLimit: 3
+      template:
+        metadata:
+          labels:
+            app: refresh_creds
+        spec:
+          serviceAccountName: edge-endpoint-service-account
+          containers:
+          - name: ecr
+            image: 767397850842.dkr.ecr.us-west-2.amazonaws.com/edge-endpoint:${IMAGE_TAG}
```

Comment:
Do you want to parameterize account and region here like you did above? Seems like we should be consistent on that one.